ui/apps/platform/cypress/integration/violations/violations.test.js

```diff
@@ -154,117 +123,81 @@ describe('Violations page', () => {
             .should('exist');
```

Comment:
I can't comment in the PR on all the unchanged `.should('exist')` lines above, but I can on this one, so asking here for all of them.

Aren't these `.should('exist')` assertions redundant ? 
The `.get` failing should suffice as the assertion in each case.

I know this is not related to this PR, but wanted to ask while I was thinking about it.